recipes/default.rb

```diff
@@ -8,7 +8,7 @@
 
 # Make sure w32time service is running
 windows_service 'w32time' do
-  run_as_user 'NT AUTHORITY\LocalService'
+  run_as_user 'NT AUTHORITY\LocalService' if ::Chef::VERSION >= 13.0
```

Comment:
In chef 12, `::Chef::VERSION` is a string you'll hit the following error:
> ArgumentError: comparison of String with 13.0 failed

You should add `.to_f` for backward compatibility.

Reply:
done